clojure-ts-mode.el

```diff
@@ -863,6 +878,27 @@ forms like deftype, defrecord, reify, proxy, etc."
         '(semantic fixed)
         clojure-ts-indent-style)))))
 
+(defun clojure-ts--docstring-fill-prefix ()
+  "The prefix string used by `clojure-ts--fill-paragraph'.
+It is simply `clojure-ts-docstring-fill-prefix-width' number of spaces."
+  (make-string clojure-ts-docstring-fill-prefix-width ? ))
+
+(defun clojure-ts--fill-paragraph (&optional justify)
+  "Like `fill-paragraph', but can handler Clojure docstrings.
+If JUSTIFY is non-nil, justify as well as fill the paragraph."
+  (let ((current-node (treesit-node-at (point))))
+    (if (clojure-ts--match-docstring nil current-node nil)
+        (let ((fill-column (or clojure-ts-docstring-fill-column fill-column))
+              (fill-prefix (clojure-ts--docstring-fill-prefix))
+              (beg-doc (treesit-node-start current-node))
+              (end-doc (treesit-node-end current-node)))
+          (save-restriction
+            (narrow-to-region beg-doc end-doc)
+            (fill-paragraph justify)))
+      (or (fill-comment-paragraph justify)
+          (fill-paragraph justify)))
+    t))
```

Comment:
Why return `t` here?

Reply:
I did it similar to the function in clojure-mode.el - https://github.com/clojure-emacs/clojure-mode/blob/master/clojure-mode.el#L754-L755 

Looking at the documentation for variable `fill-parahraph-function`:

```
Mode-specific function to fill a paragraph, or nil if there is none.
If the function returns nil, then ‘fill-paragraph’ does its normal work.
A value of t means explicitly "do nothing special".
```

Reply:
I can never remember things like these. 😅